api/src/main/java/org/pmiops/workbench/interceptors/CloudTaskInterceptor.java

```diff
@@ -22,9 +20,6 @@
 public class CloudTaskInterceptor extends HandlerInterceptorAdapter {
   public static final String QUEUE_NAME_REQUEST_HEADER = "X-AppEngine-QueueName";
   private static final String CLOUD_TASK_TAG = "cloudTask";
-  // Keep queue name consistent with the name as in OfflineRdrExportController
-  public static final Set<String> VALID_QUEUE_NAME_SET =
```

Comment:
As we're adding more task queues, this is a pain to keep synchronized with 3 other places. This validation is also unnecessary: it is sufficient to verify that the special key has been passed in the headers (only Cloud Tasks can send this). Checking the header value is informational only.